pages/apiPages/leader_page.ts

```diff
@@ -20,4 +20,23 @@ export class ApiLeadersPage {
         expect(apiRequest.ok()).toBeTruthy()
         console.log(`Leaders for period: ${period} is dispalyed`)
     }
+
+    async getMonthLeders(url: string, userToken: string ) {
```

Comment:
@PikachuSerg1 we already have getLeders method on the ApiLeadersPage for testing this endpoint,  please modify it and add proper assertions